src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -168,13 +168,26 @@ export default defineConfig({
 });
 ```
 
-:::caution
-This adapter does not support the `image.domains` and `image.remotePatterns` config properties in your Astro config. To [specify remote paths for Netlify Image CDN](https://docs.netlify.com/image-cdn/overview/#remote-path), use the `remote_images` field in `netlify.toml`.
-:::
+If you are using images hosted on another domain, you must authorize the domain or URL patterns using the [`image.domains`](/en/reference/configuration-reference/#imagedomains) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns) options:
+
+```js title="astro.config.mjs" ins={8-10}
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify';
+
+export default defineConfig({
+    // ...
+    output: 'server',
+    adapter: netlify(),
+    image: {
+      domains: ['example.com'],
+    },
+});
+```
+For more information, see [the guide to authorizing remote images](/en/guides/images/#authorizing-remote-images/). This is not required for images hosted on the same domain as your site.
 
 ### Static sites & Redirects
```

Comment:
```suggestion
### Static sites with the Netlify Adapter
```
Now that this section is about MORE than redirects, what do we think about generalizing the section?